src/hyp3_isce2/insar_tops.py

```diff
@@ -49,12 +48,9 @@ def insar_tops(
     aux_cal_dir = Path('aux_cal')
     dem_dir = Path('dem')
 
-    if download:
```

Comment:
Should the `download: bool = True` param be removed from the `insar_tops` function?

Reply:
Ack, you're right. Removed.